config/opal_configure_options.m4

```diff
@@ -544,9 +544,22 @@ fi
 AC_DEFINE_UNQUOTED([OPAL_ENABLE_GETPWUID], [$opal_want_getpwuid],
                    [Disable getpwuid support (default: enabled)])
 
-dnl We no longer support the old OPAL_ENABLE_PROGRESS_THREADS.  At
-dnl some point, this should die.
-AC_DEFINE([OPAL_ENABLE_PROGRESS_THREADS],
-          [0],
-          [Whether we want BTL progress threads enabled])
+#
+# Enable progress threads
+#
+
+AC_MSG_CHECKING([if want asynchronous progress threads])
```

Comment:
There is no benefit of having this define. Build it in, but only selectively enable it.

Reply:
default now is enabled